camel-route-support/src/main/java/io/kaoto/backend/api/service/deployment/generator/camelroute/IntegrationDeploymentGeneratorService.java

```diff
@@ -107,6 +121,12 @@ public boolean appliesTo(final List<Step> steps) {
                         .anyMatch(Predicate.isEqual(s.getKind().toUpperCase())));
     }
 
+    @Override
+    public boolean appliesToFlows(List<StepParserService.ParseResult<Step>> flows) {
```

Comment:
If one of the flows in the list of flows is from this DSL we **_assume_** that we can handle it.